examples/btc-to-evm-fungible-transfer/README.md

```diff
@@ -0,0 +1,122 @@
+# Sygma SDK BTC to EVM example
+
+## Sygma SDK ERC20 Example
+
+This is an example script that demonstrates the functionality of the SDK using the Sygma ecosystem. The script showcases a asset transfer between a BTC testnet account and a EVM Sepolia account.
```

Comment:
This is an example that demonstrates functionality of the protocol using Sygma SDK. The `src/transfer.ts` script showcases how bitcoins can transferred over to an EVM Sepolia address utilizing `@buildwithsygma/bitcoin` package.